src/__tests__/__fixtures__/sampleFlawRequired.json

```diff
@@ -1,14 +1,14 @@
 {
     "uuid": "3ede0314-a6c5-4462-bcf3-b034a15cf106",
-    "cve_id": "CVE-0000-0001",
+    "cve_id": "CVE-2024-0001",
     "impact": "MODERATE",
     "components": [
         "kernel"
     ],
     "title": "Sample title",
     "trackers": [],
     "comment_zero": "Comment Zero == Patient Zero",
-    "cve_description": "",
+    "cve_description": "Some description",
```

Comment:
Why was this added?

Reply:
the `requires_cve_description` field has the value `APPROVED`, and that only can happen if the description is not empty, another solution could be to change the value of the `requires_cve_description`

Reply:
Oh good catch. I'd prefer that second approach, removing the requires state, just cause `cve_description` is not a required field.

Reply:
If the real-world workflow is that a CVE scoring description is required, then maybe this good to add some example description